src/Compilers/CSharp/Portable/CodeGen/Optimizer.cs

```diff
@@ -972,15 +977,6 @@ public override BoundNode VisitAssignmentOperator(BoundAssignmentOperator node)
 
                 LocalSymbol localSymbol = assignmentLocal.LocalSymbol;
 
-                // If the LHS is a readonly ref and the result is used later we cannot stack
```

Comment:
>VisitSwitchDispatch [](start = 34, length = 19)

Consider moving to where `VisitSwitchStatement` was so it's clear what has changed.

Reply:
Resolved in [https://github.com/dotnet/roslyn/pull/25662](https://github.com/dotnet/roslyn/pull/25662) or earlier.

---
In reply to: [176254341](https://github.com/dotnet/roslyn/pull/25544#discussion_r176254341) [](ancestors = 176254341)